.dagger/cli.go

```diff
@@ -79,7 +79,7 @@ func (cli *CLI) Publish(
 	}
 
 	args := []string{"release", "--clean", "--skip=validate", "--verbose"}
-	if tag != "" {
+	if tag != "" && semver.Prerelease(tag) == "" {
```

Comment:
`Prerelease` is a nice touch, I was going to check if this functionality exists - great to see you had the same idea & already implemented it!